src/webui/www/public/statistics.html

```diff
@@ -1,4 +1,4 @@
-<h3>User Statistics</h3>
+<h3>QBT_TR(User Statistics)QBT_TR[CONTEXT=StatsDialog]</h3>
```

Comment:
Another issue not related to the translations is:
1. Run the app using the `--profile` switch to get a clean config
2. Open the WebUI and open the Statistics dialog. Most of the time it comes out empty without any strings apart from the headers. Sometimes it gets unstuck and loads the data.

Is anyone able to reproduce?